src/GregClient/Requests/RequestBody.cs

```diff
@@ -1,14 +1,12 @@
-﻿using RestSharp.Serialization.Json;
+﻿using System.Text.Json;
 
 namespace Greg.Requests
 {
     public abstract class RequestBody
     {
-        public static JsonSerializer jsonSerializer = new JsonSerializer();
-
         public virtual string AsJson()
         {
-            return jsonSerializer.Serialize(this);
+            return JsonSerializer.Serialize<object>(this);
```

Comment:
this is required in system.text.json for polymorphic deserialization, kind of interesting from a security standpoint.